.editorconfig

```diff
@@ -1,15 +1,9 @@
-[*.py]
-indent_style = space
-indent_size = 4
-
-[*.js]
-indent_style = space
-indent_size = 4
-
-[*.css]
-indent_style = space
-indent_size = 4
+root = true
```

Comment:
From editorconfig page
root ; ... Set to "true" to stop .editorconfig files search on current file.